DataFormats/L1THGCal/interface/HGCFETriggerDigi.h

```diff
@@ -51,7 +51,7 @@ namespace l1t {
     template<typename IDTYPE>
     IDTYPE getDetId() const { return IDTYPE(detid_); }
     template<typename IDTYPE>
-    void   setDetId(const IDTYPE& id) { detid_ = id.uint32_t(); }
+    void   setDetId(const IDTYPE& id) { detid_ = id(); }
```

Comment:
Function call doesn't compile:
`DataFormats/L1THGCal/interface/HGCFETriggerDigi.h:54:41: error: 'const class HGCTriggerDetId' has no member named 'uint32_t'`
`uint32_t(id)` and `(uint32_t)id` are working.